test/testcafe/spec/Identify_spec.js

```diff
@@ -372,6 +390,40 @@ test.requestHooks(identifyRequestLogger, identifyMockWithFingerprint)('should co
   await t.expect(factorReqHeaders['x-device-fingerprint']).eql('mock-device-fingerprint');
 });
 
+test.requestHooks(identifyRequestLogger, deviceFingerprintRequestLogger, identifyMockWithFailedFingerprint)('should proceed even if device fingerprint fails', async t => {
+  const identityPage = await setup(t, {
+    features: {
+      deviceFingerprinting: true,
+    }
+  });
+  await checkA11y(t);
+
+  await identityPage.fillIdentifierField('Test Identifier');
+
+  await identityPage.clickNextButton();
+  if (!userVariables.gen3) {
```

Comment:
In Gen3 'Next' button is being disabled immediately after click, so `clickNextButton()` can't be called twice

Reply:
Can you instead perform an assertion check to show that the button is disabled when Gen 3 is active? 

On the surface (looking only at the test), it just looks like this bug exists in Gen 3 and is being skipped/ignored.